website/content/docs/job-specification/group.mdx

```diff
@@ -63,6 +63,20 @@ job "docs" {
   rescheduling strategy. Nomad will then attempt to schedule the task on another
   node if any of the group allocation statuses become "failed".
 
+- `avoid_reschedule_on_lost` `(bool: false)` - Defines the reschedule behaviour 
+  of an allocation when the node it is running on becomes disconnect or lost. 
+  When enabled, if the node it is running on becomes disconnected 
+  or goes down, this allocations wont be rescheduled and will show up as `unknown`
+  until the node comes back up or it is manually restarted. 
+  This behaviour will only modify the reschedule process on the server.
+  To modify the allocation behaviour on the client, see `stop_after_client_disconnect`.
+
+  Setting `max_client_disconnect` and `avoid_reschedule_on_list=true` at the same time 
+  requires that [rescheduling is disabled entirely][] If [`max_client_disconnect`](#max_client_disconnect) 
+  is set and `avoid_reschedule_on_lost=true`, allocations on disconnected nodes will be
+  `unknown` until the `max_client_disconnect` window expires, at which point they 
+  become `lost` but will not be rescheduled. 
```

Comment:
Notes from our sidebar conversation:
* We'll need to correct this line.
* And add some documentation on how to recover the unknown allocation if you want (maybe just re-run the job)
* Once those are done this should be LGTM!